fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/impl/registry/sync/RegistryAttributesImpl.java

```diff
@@ -0,0 +1,33 @@
+package net.fabricmc.fabric.impl.registry.sync;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import net.minecraft.util.registry.Registry;
+
+import net.fabricmc.fabric.api.event.registry.RegistryAttribute;
+import net.fabricmc.fabric.api.event.registry.RegistryAttributeHolder;
+
+public class RegistryAttributesImpl<T> implements RegistryAttributeHolder<T> {
+	private static final Map<Registry<?>, RegistryAttributeHolder<?>> registryAttributeMap = new HashMap<>();
```

Comment:
Would storing the values or implementing the `RegistryAttributeHolder` onto the Registry make more sense?

Reply:
Yes, possibly. I dont see why not